### DISCUSSION TITLE
Tags question

### DISCUSSION THREAD

**Discussion by davidjdriver** [vkbo/novelWriter#647] [Help] [answered]
I am giving novelWriter a try on Ubuntu 20.04. I have the setup working. I launched the first time and I generated the big demo document so I have something to experiment with. I went into one of the chapters and added tags for @pov and @char for characters bubba and bubba2. I then created notes in the Characters folder titled bubba and bubba2. I just added a sentence to each note to have something in the note. I went back to the chapter and the two names in the pov/char tags are underlined with green squigglies and ctrl+enter gives the error "Could not find the reference for tag 'bubba'. It either doesn't exist, or the index is out of date. The index can be updated from the Tools menu, or by pressing F9." The docs say green squigglies mean it can't find the related tag.  I hit F9 to reindex the documents and still have the squiggles and get the same message when hitting ctrl+enter. 

Is there somthing I have to do to the notes under the Characters folder to make the navigation work? I have read through the documentation at https://novelwriter.readthedocs.io/ and I am sure that I am just missing what I need to do but I can't find it.

**Comment by vkbo** [comment, marked as answer]
Hi!

I think I know what the problem is. The reference points to a tag, and the green squiggly line indicates that the tag does not exist. Note files need to have a tag set to be able to point to them. Have a look at: https://novelwriter.readthedocs.io/en/stable/usage_notes.html#tags-in-notes

**Reply by vkbo** [reply]
The references and tags are described in different sections of the documentation. I'll add cross-references between those two section to make it easier to connect them while reading the docs.

**Reply by davidjdriver** [reply]
That is precisely the problem. I am not sure how I missed that section when I got the section mentioning the squiggly lines meaning it can't find the tag. 

Thanks!
